site/docs-md/guides/ionic-framework-app.md

```diff
@@ -0,0 +1,250 @@
+  photo: SafeResourceUrl;
+
+  constructor(private sanitizer: DomSanitizer) {  }
+
+  async takePicture() {
+    const image = await Plugins.Camera.getPhoto({
+      quality: 100,
+      allowEditing: false,
+      resultType: CameraResultType.Base64,
+      source: CameraSource.Camera
+    });
+
+    this.photo = this.sanitizer.bypassSecurityTrustResourceUrl(image && (image.base64Data));
+  }
+}
+```
+
+Let's give this a try in the browser:
+
+```shell
+ionic serve
+```
+
+Open the Developer Tools and select the `Console` tab. Then, navigate over to Tab Two, then click the newly created Camera fab button. An error appears in the Console:
+
+`ERROR Error: Uncaught (in promise): TypeError: cameraModal.present is not a function`
+
+Some Capacitor plugins, including the Camera, provide the web-based functionality and UI via the Ionic [PWA Elements library](https://github.com/ionic-team/ionic-pwa-elements). It's a separate dependency, so install it using the Terminal (cancel the `ionic serve` command currently running first):
+
+```shell
+npm install @ionic/pwa-elements
+```
+
+**Note:** There's currently an issue with importing the Ionic PWA elements library in an Ionic Angular app. In the meantime, include this script tag in `src/index.html`: 
+
+```html
+<head>
+  <!-- Other tags -->
+
+  <script src="https://unpkg.com/@ionic/pwa-elements@1.0.2/dist/ionicpwaelements.js"></script>
+</head>
+```
+
+With that appropriately configured now, re-run `ionic serve` and click the Camera fab button. If your computer has a webcam of any sort, a modal window displays and you can take a photo!
+
+![Camera API on the web](/assets/img/docs/guides/ionic-framework/camera-web.png)
+*"I'm pretty handsome." - The Author*
+
+Next up, let's run this app on iOS and Android.
+
+## iOS
+Capacitor iOS apps are configured and managed through Xcode, with dependencies managed by CocoaPods. Before running this app on an iOS device, there's a couple of steps to complete.
+
+From the Terminal, run the Capacitor `sync` command, which updates all native code dependencies as well as copies all web assets (the Ionic Angular app in this case) into the native iOS project:
+
+```shell
+npx cap sync
+```
+
+Next, run the Capacitor `open` command, which opens the native iOS project in Xcode:
+
+```shell
+npx cap open ios
+```
+
+Within Xcode, click on `App` in the Project Navigator on the left-hand side, then within the `Signing` section, select your Development Team. 
+
+![Xcode - Selecting Development Team](/assets/img/docs/guides/ionic-framework/xcode-signing.png)
+
+Next, for the Camera plugin to work, we must configure the "Privacy - Camera Usage" permission. iOS displays a modal dialog automatically after the first time that `Camera.getPhoto()` is called, prompting the user to allow the app to use the Camera. To set this, the `Info.plist` file must be modified ([more details here](../ios/configuration/)). To access it, click "Info," then expand "Custom iOS Target Properties."
```

Comment:
We already add a default text, so it's not really needed to add it, just change it in case you don't like it

Reply:
yep.  the idea is to get the developer following this to be more comfortable working in Xcode/making changes.